### ISSUE TITLE
Where to ask questions ?

### ISSUE THREAD

**Issue by jwende** [open-horizon/examples#50]
The discourse server (https://discourse.bluehorizon.network/) does not allow any new entries .
How may we ask questions or report problems ?

**Comment by jwende** [comment]
thanks - but in this case you might update the links in the documentation.
J.

**Comment by TheMosquito** [comment]
Yes, and actually I should have thanked you for raising this issue because it made me do exactly that immediately after responding to this issue.  The Discourse site used to point to the Watson IoT Platform site, which itself had a pointer to the question space in its first paragraph.  But after your comment I realized that was a bit hidden, so I also added that question space link to the Discourse banner.  Hopefully that will help others find help when they need it.  Thanks!
